cni/pkg/taint/taint_test_helper.go

```diff
@@ -0,0 +1,426 @@
+package taint
+
+import (
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type makeConfigMapArgs struct {
+	ConfigName  string
+	Namespace   string
+	Labels      map[string]string
+	Annotations map[string]string
+	Data        map[string]string
+}
+
+const (
+	ValidationContainerName = "istio-validation"
+)
+
+func makeConfigMap(args makeConfigMapArgs) *v1.ConfigMap {
```

Comment:
Do you need to return a pointer? since it looks like you always dereference it anyways